snips_nlu/slot_filler/feature_factory.py

```diff
@@ -27,6 +29,22 @@
 from snips_nlu.slot_filler.features_utils import (
     entity_filter, get_word_chunk, initial_string_from_tokens)
 
+logger = logging.getLogger(__name__)
+
+class _InvalidCustomEntityFilter(ValueError):
+    pass
+
+
+CUSTOM_ENTITIES_FILTER_KEYS = {"automatically_extensible"}
+
+
+def _check_custom_entity_filter(entity_filter):
+    for k in entity_filter:
+        if k not in CUSTOM_ENTITIES_FILTER_KEYS:
+            msg = "Invalid custom entity filter key '%s'. Accepted filter " \
+                  "keys are %s" % (k, list(CUSTOM_ENTITIES_FILTER_KEYS))
```

Comment:
Besides, IMO this part of the code (from `_InvalidCustomEntityFilter`) would be better placed right after the `CustomEntityMatchFactory`.

Reply:
I move the function but didn't automate the exception message since the exception is meant to be raise for any kind of invalid filter, not only for invalid keys. If we add more complex filters in the future, we might want to check the filter values and and raise the exact same exception